yarn.lock

```diff
@@ -2729,6 +2866,11 @@ has@^1.0.3:
   dependencies:
     function-bind "^1.1.1"
 
+helmet@^4.0.0:
```

Comment:
its weird helmet has changed here

Reply:
It's because @types/koa-helmet is looking for `helmet@^4.0.0:` which resolves to a different thing than what we had for `helmet@^4.1.1`. If we run a subdeps update like `yarn upgrade --all` that would make them both resolve to 4.4.1, but I tried that recently and we can't update subdeps until we fix our types - it makes our tests fail

Reply:
I selectively updated the subdeps to make that cleaner